src/components/Calculator.js

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import './Calculator.css';
+
+function Calculator() {
+  const handleClick = () => {
+    // Handle click logic here
+  };
+
+  return (
+    <div className="calculator">
+      <div className="display">0</div>
+      <div className="buttons">
+        <div className="button-row">
+          <button type="button" className="button function-button">AC</button>
+          <button type="button" className="button function-button">+/-</button>
+          <button type="button" className="button function-button">%</button>
+          <button type="button" className="button operator-button">÷</button>
+        </div>
+        <div className="button-row">
+          <button type="button" className="button" onClick={() => handleClick(7)}>7</button>
+          <button type="button" className="button" onClick={() => handleClick(8)}>8</button>
+          <button type="button" className="button" onClick={() => handleClick(9)}>9</button>
+          <button type="button" className="button operator-button">x</button>
+        </div>
+        <div className="button-row">
+          <button type="button" className="button" onClick={() => handleClick(4)}>4</button>
+          <button type="button" className="button" onClick={() => handleClick(5)}>5</button>
+          <button type="button" className="button" onClick={() => handleClick(6)}>6</button>
+          <button type="button" className="button operator-button">-</button>
+        </div>
+        <div className="button-row">
+          <button type="button" className="button" onClick={() => handleClick(1)}>1</button>
+          <button type="button" className="button" onClick={() => handleClick(2)}>2</button>
+          <button type="button" className="button" onClick={() => handleClick(3)}>3</button>
+          <button type="button" className="button operator-button">+</button>
+        </div>
+        <div className="button-row">
+          <button type="button" className="button" onClick={() => handleClick(0)}>0</button>
+          <button type="button" className="button" onClick={() => handleClick('.')}>.</button>
+          <button type="button" className="button operator-button">=</button>
```

Comment:
- [ ] Kindly make sure that the design match the given design template, as it is required in the style guideline 👍 

Your design 

![Capture](https://github.com/AlyAtalla/math-magician/assets/80458199/440288c1-eb10-4dd0-bdf7-6b7c461a205d)

Design to match

![Capture1](https://github.com/AlyAtalla/math-magician/assets/80458199/650e9cb0-5ad9-4948-af27-8978fe06e457)

Reply:
thank you